GameWindow.java

```diff
@@ -38,25 +38,30 @@
 	public final String TITLE = "Melee Tech Trainer";
 
 	private static SaveFile hs;
-
+	
 	private boolean running = false;
 	private Thread thread;
 
 	// sound files
 	private File aSound = new File("./Notes/A.wav");
 	private File cSound = new File("./Notes/C.wav");
-
+	private File eSound = new File("./Notes/E.wav");
+	private File gsSound = new File("./Notes/Gsharp.wav");
+	
+	
+	
+	
+	
 	private int pause;
-	private int score;
-	// int mx = 0;
-	// int my = 0;
-
-	private BufferedImage currentBackground = null;
+	private int score = 0;
+	private int beginnerScores[] = new int[10];
+	private int intermediateScores[] = new int[10];
+	private int advancedScores[] = new int[10];

```

Comment:
This is another good opportunity for a map or an array, as are many other related variable definitions down below. I won't call out every single one, but most times when you have a bunch of things which you really want to treat as a group (e.g. all of your notes below) they should be in a collection rather than being separate variables.

Reply:
using a map for our button flashes also caused the button releases to not register where when a button was pressed, the button would turn white and stay white until another button is pressed.

Reply:
That shouldn't be effected by how you're storing them (individual buttons vs a map), so it's probably just a matter of a bug to sort out somewhere or other.